src/docs/goose-book/src/getting-started/validation.md

```diff
@@ -0,0 +1,33 @@
+# Validation
+
+## Goose Eggs
+[Goose-eggs](https://github.com/tag1consulting/goose-eggs) is a validation library that can be used with Goose. 
+
+Include the goose-eggs crate in the dependency section of your `Cargo.toml`. 
+
+```toml
+[dependencies]
+goose-eggs = "0.4.0"
```

Comment:
I usually drop the trailing `0`, so just `0.4` to better indicate that newer releases in the `0.4.x` release series will be automatically updated when you run `cargo update`.